bundler/lib/dependabot/bundler/file_updater/ruby_requirement_setter.rb

```diff
@@ -49,7 +50,11 @@ def declares_ruby_version?(node)
         end
 
         def ruby_version
-          requirement = Gem::Requirement.new(ruby_requirement)
+          requirement = if !ruby_requirement.is_a?(Gem::Requirement)
+                          Bundler::Requirement.new(ruby_requirement)
```

Comment:
Let's make it super clear that this is not `::Bundler`?

```suggestion
                          Dependabot::Bundler::Requirement.new(ruby_requirement)
```

Alternatively we could drop the namespaces altogether since we're already in `Dependabot::Bundler`, but I prefer being explicit here

Reply:
Yes good point!